src/rules/jsx_no_unescaped_entities.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+use super::{Context, LintRule};
+use crate::diagnostic::{LintFix, LintFixChange};
+use crate::handler::{Handler, Traverse};
+use crate::Program;
+use deno_ast::view::{JSXElement, JSXElementChild};
+use deno_ast::SourceRanged;
+
+#[derive(Debug)]
+pub struct JSXNoUnescapedEntities;
+
+const CODE: &str = "jsx-no-unescaped-entities";
+
+impl LintRule for JSXNoUnescapedEntities {
+  fn tags(&self) -> &'static [&'static str] {
+    &["react", "jsx"]
+  }
+
+  fn code(&self) -> &'static str {
+    CODE
+  }
+
+  fn lint_program_with_ast_view(
+    &self,
+    context: &mut Context,
+    program: Program,
+  ) {
+    JSXNoUnescapedEntitiesHandler.traverse(program, context);
+  }
+
+  #[cfg(feature = "docs")]
+  fn docs(&self) -> &'static str {
+    include_str!("../../docs/rules/jsx_no_unescaped_entities.md")
+  }
+}
+
+const MESSAGE: &str = "Found one or more unescaped entities in JSX text";
+const HINT: &str = "Escape the '\">} characters respectively";
```

Comment:
This is confusing - is `'` character supposed to be escaped? If so it's not listed in the list below

Reply:
It's listed here https://github.com/denoland/deno_lint/blob/a2a2bea499fa44d26ad29846866eca039741223d/src/rules/jsx_no_unescaped_entities.rs#L52